package.json

```diff
@@ -58,9 +58,10 @@
     "mkdirp": "^0.5.1",
     "mockery": "^2.1.0",
     "preact": "^10.1.1",
-    "preact-i18n": "^1.4.1",
+    "preact-context-provider": "^2.0.0-preactx.2",
+    "preact-i18n": "^2.0.0-preactx.2",
     "preact-render-to-string": "^5.1.3",
-    "preact-router": "^3.1.0",
+    "preact-router": "github:zimbra/preact-router#3.1.0_base_path_support",
```

Comment:
I'm pretty sure we can remove these dependencies entirely, since we don't actually depend on them. They are injected by the main app, so they are not maintained by the package manager.

I can do that separate from this ticket though since it is an existing issue.

Reply:
I was unable to use the router in the zimlet without this. Maybe I was doing something wrong?